src/BlazorElectron/BlazorElectron.csproj

```diff
@@ -1,8 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-	<RuntimeIdentifier>win-x64</RuntimeIdentifier>
+    <!--
+    In some systems the folder/configuration win-x64 can cause problems (especially when reading/writing the Excel File) due security settings, anti-virus and Co.
+    When you need to compile to Electron App, you must uncomment the RuntimeIdentifier.
+    If the folder/configuration win-x64 does not cause any problems for you, you also can uncomment this line.
+    YOU MUST do this ON ALL '.csproj' files
+    -->
+    <!--
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
Das ist nicht so cool! Schau mal dass du dein Antivierenporgramm anders einstellst. Zur Not kann man auch contitions machen:

`<PropertyGroup Condition="'$(Configuration)'=='Release'">
    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
</PropertyGroup>`

Probier da mal noch bisschen was aus!